src/main/java/uk/gov/companieshouse/company/profile/controller/CompanyProfileController.java

```diff
@@ -75,7 +75,10 @@ public ResponseEntity<Void> addExemptionsLink(
             @PathVariable("company_number") String companyNumber) {
         logger.info(String.format("Payload successfully received on PATCH endpoint "
                 + "with contextId %s and company number %s", contextId, companyNumber));
-        companyProfileService.addExemptionsLink(contextId, companyNumber);
+
+        String linkType = "exemptions";
+        String deltaType = "exemption_delta";
+        companyProfileService.addExemptionsLink(contextId, companyNumber, linkType, deltaType);
```

Comment:
The parameters for addExemptionsLink could be encapsulated within some kind of link request object.